gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -64,10 +64,10 @@ def get_shared_images
   end
 
   def get_public_images
+    filters = @options.public_images_filters
```

Comment:
hmmm, having hard time testing it, the chain of calls is deep, and can't seem to use [this](https://www.relishapp.com/rspec/rspec-mocks/v/2-0/docs/stubs/stub-a-chain-of-methods) to simplify things